source/grid/manifold_lib.cc

```diff
@@ -2225,29 +2217,21 @@ TransfiniteInterpolationManifold<dim, spacedim>::
   // The methods to identify cells around points in GridTools are all written
   // for the active cells, but we are here looking at some cells at the coarse
   // level.
-  Assert(triangulation != nullptr, ExcNotInitialized());
-  Assert(triangulation->begin_active()->level() >= level_coarse,
-         ExcMessage("The manifold was initialized with level " +
-                    std::to_string(level_coarse) + " but there are now" +
-                    "active cells on a lower level. Coarsening the mesh is " +
-                    "currently not supported"));
+
+  // FIXME IMPROVE
```

Comment:
@kronbichler I am not sure whether we can actually speed up the coarse cell loop below in a meaningfully way - we actually loop over all coarse cells in order to construct a full list of candidate cells.

Reply:
We certainly could by creating bounding boxes (or even an RTree) that allows us to identify a (small list) of cells as candidates for a given set of points. That would help tremendously in case we have many coarse mesh cells.